go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	k8s.io/klog v1.0.0
 	k8s.io/kube-aggregator v0.17.6
 	k8s.io/kube-openapi v0.0.0-20200410145947-bcb3869e6f29
+	k8s.io/kubernetes v1.14.0
```

Comment:
I had a comment for another PR can apply here: https://github.com/vmware-tanzu/antrea/pull/772#discussion_r441312144.
I think it's easier for this PR to drop the dependency on "k8s.io/kubernetes"